### DISCUSSION TITLE
Hi, I need to make Filament\Forms\Components\Multiselect so that each option can be deleted.

### DISCUSSION THREAD

**Discussion by Narek1331** [filamentphp/filament#14352] [Help]
### Package

Form builder

### Package Version

v3.0.0

### How can we help you?

Hi, I need to make Filament\Forms\Components\Multiselect so that each option can be deleted.
![Screenshot from 2024-09-26 16-20-26](https://github.com/user-attachments/assets/751854ae-7b70-46e4-88b2-18e4dac5cc64)

**Comment by leandrocfe** [comment]
Sorry, what does it mean?
In v3 is [multiple](https://filamentphp.com/docs/3.x/forms/fields/select#multi-select)
